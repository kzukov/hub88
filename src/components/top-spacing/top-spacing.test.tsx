import TopSpacing from "./top-spacing";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("TopSpacing", () => {
  test("Snapshot", () => {
    const { asFragment } = render(<TopSpacing>children</TopSpacing>);

    expect(asFragment()).toMatchSnapshot();
  });

  test("Structure", () => {
    const testChildren = "Test children";

    const { container } = render(<TopSpacing>{testChildren}</TopSpacing>);

    expect(container.firstChild).toHaveClass(`mt-3`);
  });
});
