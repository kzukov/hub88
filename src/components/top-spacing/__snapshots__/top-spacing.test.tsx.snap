// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopSpacing Snapshot 1`] = `
<DocumentFragment>
  <div
    class="mt-3"
  >
    children
  </div>
</DocumentFragment>
`;
